python/lsst/ci/prepare.py

```diff
@@ -554,8 +565,8 @@ def __getBuildId(self, manifest, manifestSha):
                 git = Git(self.dbdir)
                 tags = git.tag('-l', 'b[0-9]*').split()
                 btre = re.compile('^b[0-9]+$')
-                btags = [ 0 ]
-                btags += [ int(tag[1:]) for tag in tags if btre.match(tag) ]
+                btags = [0]
+                btags += [int(tag[1:]) for tag in tags if btre.match(tag)]
```

Comment:
The `tag` variable in this list comprehension overwrites the value defined from line 559 above.